examples/tcp-close/main.rs

```diff
@@ -20,12 +20,13 @@ use crate::{
     client::TcpClient,
     server::TcpServer,
 };
-use ::std::time::Duration;
-use anyhow::Result;
-use demikernel::{
+use ::demikernel::{
+    timer,
     LibOS,
     LibOSName,
 };
+use ::std::time::Duration;
+use anyhow::Result;
```

Comment:
Since you touched this line could you fix it and prefix it with `::`?

Reply:
Done.